src/utils/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 LOCK = threading.Lock()
 
-MERGEABLE_SETTINGS = {"INSTALLED_APPS", "MIDDLEWARE_CLASSES"}
+MERGEABLE_SETTINGS = {}
```

Comment:
We try to be extra defensive when removing public constants like this in case a plugin out there might be using them, so we keep them for at least one release cycle.

Reply:
That makes sense :-). For full backwards-compatibility then, should this be unchanged, and be the default if it's not provided in `custom_settings`?

Reply:
Oh, though as `{}` it's now a dictionary and not a set as before :-)